### PR TITLE
GUI: Show how to pull an image using podman pull (PROJQUAY-3948)

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

I don't know how to render this to make sure it looks ok, could someone check to see how this renders?

### CODE REVIEW

**Comment by dmage** [comment]
When builds are disabled:
Screen Shot 2022-05-19 at 9 25 03

When builds are enabled:
Screen Shot 2022-05-19 at 9 26 09

**Comment by dmage** [comment]
@dmesser what do you think UI-wise?

**Comment by dmesser** [comment]
@dmage IMHO putting some icons next to the pull commands would be a nice touch. In the second screenshot the two text fields should be ordered vertically instead of sitting next to each other horizontally.

File: static/directives/repo-view/repo-panel-info.html [hunk 1]
```diff
@@ -23,6 +23,12 @@
     </div>
 
     <!-- Builds -->
+    <div class="col-sm-7 builds-list" ng-if="!Features.BUILD_SUPPORT">
```
Comment (dmage) [inline static/directives/repo-view/repo-panel-info.html:26]: You can remove `builds-list` from this element (and from the next one for docker). It'll decrease its height.

**Comment by rhatdan** [comment]
What do I need to do to move this forward?

**Review by dmage — APPROVED** [review]

**Pull request merged**